README.md

```diff
@@ -2257,6 +2260,248 @@ for (CreditPreview preview : previews){
 }
 ```
 
+## Identity
+
+### Create a IndividualIdentity
+You can create a IndividualIdentity for validate natural person documents
+
+```java
+import com.starkinfra.*;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+
+HashMap<String, Object> identity = new HashMap<>();
+identity.put("name", "Walter White");
+identity.put("taxId", "012.345.678-90");
+identity.put("tags", new String[]{"breaking", "bad"});
+
+List<IndividualIdentity> identities = new ArrayList<>();
+identities.add(new IndividualIdentity(identity));
+
+List<IndividualIdentity> identities = IndividualIdentity.create(identities);
+
+for (IndividualIdentity identity : identities){
+    System.out.println(identity);
+}
+```
+
+**Note**: Instead of using IndividualIdentity objects, you can also pass each element in dictionary format
+
+### Query IndividualIdentity
+
+You can query multiple individual identities according to filters.
+
+```java
+import com.starkinfra.IndividualIdentity;
+import com.starkinfra.utils.Generator;
+
+HashMap<String, Object> params = new HashMap<>();
+params.put("limit", 3);
+params.put("status", "success");
+params.put("after", "2019-04-01");
+params.put("before", "2030-04-30");
+
+Generator<IndividualIdentity> identities = IndividualIdentity.query(params);
+
+for (IndividualIdentity identity : identities) {
+    System.out.println(identity);
+}
+```
+
+### Get a IndividualIdentity
+
+After its creation, information on a individual identity may be retrieved by its id.
+
+```java
+import com.starkinfra.IndividualIdentity;
+
+IndividualIdentity identity = IndividualIdentity.get("5155165527080960");
+
+System.out.println(identity);
+```
+
+### Update an IndividualIdentity
+
+You can update a specific identity status to send it to validation.
+
+```java
+import com.starkinfra.IndividualIdentity;
+
+IndividualIdentity identity = IndividualIdentity.update("5155165527080960", "processing")
+
+System.out.println(identity);
+```
+
+**Note**: For apply it to processing status, you should send requireds IndividualDocuments
```

Comment:
**Note**: For apply it to processing status, you should send required IndividualDocuments